source/Jacwright/RestServer/RestServer.php

```diff
@@ -372,6 +380,9 @@ protected function generateMap($class, $basePath) {
 	}
 
 	public function getPath() {
+		$getParameters = parse_url ($_SERVER['REQUEST_URI'], PHP_URL_QUERY);
+		parse_str($getParameters, $this->getParamAssoc);
```

Comment:
Not sure why this is necessary, I'll do some tests to double-check, but iirc `$_GET` worked fine.